apps/samradsgatt/components/Layout/Layout.tsx

```diff
@@ -1,3 +1,4 @@
+import { GridContainer } from '@island.is/island-ui/core'
```

Comment:
það er að koma warning hér :)

'GridContainer' is defined but never used. Allowed unused vars must match /^_[a-zA-Z0-9]*$/u